test/source/mock/google/strategies/send-message-strategy.ts

```diff
@@ -17,10 +17,22 @@ class SaveMessageInStorageStrategy implements ITestMsgStrategy {
   }
 }
 
-class PwdEncryptedMessageTestStrategy implements ITestMsgStrategy {
+class PwdEncryptedMessageWithFlowCryptComApiTestStrategy implements ITestMsgStrategy {
   public test = async (mimeMsg: ParsedMail, base64Msg: string) => {
     if (!mimeMsg.text?.match(/https:\/\/flowcrypt.com\/[a-z0-9A-Z]{10}/)) {
-      throw new HttpClientErr(`Error: cannot find pwd encrypted link in:\n\n${mimeMsg.text}`);
+      throw new HttpClientErr(`Error: cannot find pwd encrypted flowcrypt.com/api link in:\n\n${mimeMsg.text}`);
+    }
+    if (!mimeMsg.text?.includes('Follow this link to open it')) {
+      throw new HttpClientErr(`Error: cannot find pwd encrypted open link prompt in ${mimeMsg.text}`);
+    }
+    (await GoogleData.withInitializedData(mimeMsg.from!.value[0].address!)).storeSentMessage(mimeMsg, base64Msg);
```

Comment:
why do we store it if we don't read it afterwards?

Reply:
You're right, no point storing it here. Some tests read it back or need it afterwards, but this one can just check the message on the fly and discard it. Removing